wurst/objects/abilities/HostileAnimalAbilities.wurst

```diff
@@ -0,0 +1,79 @@
+package HostileAnimalAbilities
+
+// Standard lib Imports:
+import AbilityObjEditing
+import Assets
+import BuffObjEditing
+import ObjectIdGenerator
+import ObjectIds
+
+// Local Imports:
+import LocalObjectIDs
+import ToolTipsUtils
+
+let BEAR_BLOCK_CHANGE = 0.5
+let BEAR_BLOCK_DAMAGE_IGNORED = 7.
+// let BEAR_BLOCK_TT = "" +
+//     "This unit has hard skin; ith as {0} to reduce incoming damage by {1}"
+//     .format(
+//         BEAR_BLOCK_CHANGE.toToolTipOrange(),
+//         BEAR_BLOCK_DAMAGE_IGNORED.toToolTipOrange()
+//     )
+
+let WOLF_LIFE_STEAL_AMOUNT = 0.35
+// let WOLF_LIFE_STEAL_TT = "This unit restore {0} of his damage dealt as hit point"
+//     .format(WOLF_LIFE_STEAL_AMOUNT.toToolTipRed())
+
+
+// TODO: Display a buff indicating that the unit has a passive ability
+// Hardened skin & vampiric attack does not seems to apply a buff on the caster
+@compiletime function createBearBlock() returns AbilityDefinitionHardenedSkin
+    // let buffId = BUFF_ID_GEN.next()
+    // new BuffDefinition(buffId, 'Basl')
+    //     ..setName(1, "Bear Block")
+    //     ..setTooltipNormal(1, "Hardened Skin")
+    //     ..setTooltipNormalExtended(1, BEAR_BLOCK_TT)
+    //     ..setIcon(Icons.bTNHardenedSkin)
+
+    return new AbilityDefinitionHardenedSkin(ABILITY_BEAR_BLOCK)
+        // ..setBuffs(1, buffId.toRawCode())
+        ..setCheckDependencies(false)
+        ..setChancetoReduceDamage(1, BEAR_BLOCK_CHANGE * 100)
+        ..setMinimumDamage(1, 0.0)
+        ..setIgnoredDamage(1, BEAR_BLOCK_DAMAGE_IGNORED)
+        ..setName("Bear Block")
+
+@compiletime function createBearBash() returns AbilityDefinitionBashcreep
+    return new AbilityDefinitionBashcreep(ABILITY_BASH)
+        ..setDurationHero(1, 1.0)
+
+
+@compiletime function createPantherShadowMeld() returns AbilityDefinitionShadowMeld
+    return new AbilityDefinitionShadowMeld(ABILITY_PANTHER_SHADOW_MELD)
```

Comment:
Where is this ID defined? Is it supposed to `ABILITY_PANTHER_PROWL`?

Reply:
I removed the Id definition on merging, my mistake, this is the ability that allow panther to become invisible at night.